client/src/modules/invoices/templates/grid/status.tmpl.html

```diff
@@ -7,10 +7,10 @@
   class="ui-grid-cell-contents">
 
   <!-- warning state: invalid, but not initialised yet -->
-  <span ng-hide="row.entity._initialised" class="fa fa-circle-o" uib-tooltip="{{ row.entity._message | translate }}" tooltip-placement="right" tooltip-append-to-body="true"></span>
+  <span ng-hide="row.entity._initialised" class="fa fa-circle-o" translate-attr="{ 'uib-tooltip': 'row.entity._message' }"  tooltip-placement="right" tooltip-append-to-body="true"></span>
```

Comment:
The same observation goes for this one:  I'm not sure the `uib-tooltip` will work since it is a directive.  Additionally, the `row.entity._message` should not be wrapped in `'`.